sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 1.10.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.9.1 (2023-06-06)
 
 ### Other Changes
+#### Behavioral breaking change 
+- Moved `AzureDeveloperCliCredential` to the end of the `DefaultAzureCredential` chain.
```

Comment:
Has this change been reviewed by @srnagar or @JonathanGiles? At first glance it doesn't feel right to introduce a behavioral breaking change in a patch release.

Reply:
Yes. This shipped in 1.9. After we shipped a very late archboard decision came that changed the order, so we're catching up. @srnagar approved.